frontend/README.md

```diff
@@ -50,7 +50,38 @@ However, you may also need to test it against a deployed version of Explorer bac
 case you can use:
 
 ```
-$ npm run dev:production-wamp
+$ npm run dev:production-wamp-with-indexer-mainnet
+```
+
+Or you could run frontend and backend separately. Start frontend like this:
+```
+$ npm run dev:local-wamp-with-indexer-mainnet
+```
+
+Then you need to start WAMP server:
+
+```
+$ docker-compose up -d wamp
+```
+
+After it you need to open `backend/config/` and create a `env-indexer-mainnet-local` file 
+(or just copy `env-indexer-mainnet` and add `-local` prefix to the file name) and add/replace this strings
+(Also replace `...` with your credentials):
+
+```
+export NEAR_RPC_URL=https://archival-rpc.mainnet.near.org
+export NEAR_INDEXER_DATABASE_HOST=...
+export NEAR_INDEXER_DATABASE_NAME=mainnet_explorer
+export NEAR_INDEXER_DATABASE_USERNAME=...
+export NEAR_INDEXER_DATABASE_PASSWORD=...
+```
+Then make this:
+```
+$ source "$(dirname "$BASH_SOURCE")/env-indexer-mainnet-local"
+```
```

Comment:
No need for this step, it is included in the `start:mainnet-with-indexer` script:

```suggestion

```